src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div class="checkout-receipt">
+		<h2 class="checkout-receipt__headline">
+			Order Confirmation
+		</h2>
+		<section class="section">
+			TODO: Date<br>
+			{{ lender.firstName }} {{ lender.lastName }}<br>
+			{{ lender.email }}
+		</section>
+		<section>
+			<ul class="checkout-receipt__loan-list">
+				<li
+					class="section"
+					v-for="item in receipt.items.values"
+					:key="item.id"
+				>
+					<div
+						class="loan"
+						v-if="item.basketItemType === 'loan_reservation'"
+					>
+						<h3>
+							<router-link
+								class="loan__name"
+								:to="`/lend/${item.loan.id}`"
+							>
+								{{ item.loan.name }}
+							</router-link>
+						</h3>
+						<div class="loan__meta">
+							<p class="loan__meta-city">
+								<span v-if="item.loan.geocode && item.loan.geocode.city">
+									{{ item.loan.geocode.city }},
+								</span>
+								<span v-if="item.loan.geocode && item.loan.geocode.country">
+									{{ item.loan.geocode.country.name }}
+								</span>
+							</p>
+							<p>{{ item.loan.use }}</p>
+						</div>
+						<div class="loan__amount">
+							${{ item.price }}
+						</div>
+					</div>
+					<div
+						class="loan"
+						v-else-if="item.basketItemType === 'kiva_card'"
+					>
+						<h3 class="loan__name">
+							Kiva Card
+						</h3>
+						<div class="loan__amount">
+							${{ item.price }}
+						</div>
+					</div>
+					<div
+						class="loan"
+						v-else-if="item.basketItemType === 'donation'"
+					>
+						<h3 class="loan__name">
+							Donation
+						</h3>
+						<div class="loan__amount">
+							${{ item.price }}
+						</div>
+					</div>
+				</li>
+			</ul>
+		</section>
+		<section class="section split total">
+			<h3 class="total__header split__item">
+				Total:
+			</h3>
+			<span class="total__amount split__item split__item--end">${{ receipt.totals.itemTotal }}</span>
+		</section>
+		<section class="section payments">
+			<h2 class="payments__header">
+				Payments
```

Comment:
I think we should include all of these payment types https://github.com/kiva/kiva/blob/development/httpdocs/views/Basket/ThanksView.tpl#L163-L197. And probably in the order that they're put in there too. These totals are all available through graphql already.